package.json

```diff
@@ -49,7 +49,8 @@
     "test": "pnpm run lint && pnpm run build && pnpm run size",
     "postversion": "pnpm run build",
     "prepublishOnly": "rm -rf ./package && clean-publish",
-    "postpublish": "rm -rf ./package"
+    "postpublish": "rm -rf ./package",
+    "preinstall": "pnpm only-allow pnpm"
```

Comment:
Is there a reason why you've added this as a dependency instead of using `npx`? If the person doesn't have `pnpm` installed, wouldn't this just fail?

Reply:
You're right my bad. It was just me used to using `pnpm` to run scripts 😅